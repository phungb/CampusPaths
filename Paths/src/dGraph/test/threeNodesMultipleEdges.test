# Tests a graph of multiple nodes and edges

# Create a graph
CreateGraph graph3

# Add three nodes
AddNode graph3 n1
AddNode graph3 n2
AddNode graph3 n3

# Add edges to existing nodes
AddEdge graph3 n1 n1 e11
AddEdge graph3 n1 n2 e12
AddEdge graph3 n1 n3 e13
AddEdge graph3 n2 n1 e21
AddEdge graph3 n2 n2 e22
AddEdge graph3 n2 n3 e23
AddEdge graph3 n3 n1 e31
AddEdge graph3 n3 n2 e32
AddEdge graph3 n3 n3 e33

# Print the nodes in the graph
ListNodes graph3
ListChildren graph3 n1
ListChildren graph3 n2
ListChildren graph3 n3