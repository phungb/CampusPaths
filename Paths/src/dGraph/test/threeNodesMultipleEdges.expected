# Tests a graph of multiple nodes and edges

# Create a graph
created graph graph3

# Add three nodes
added node n1 to graph3
added node n2 to graph3
added node n3 to graph3

# Add edges to existing nodes
added edge e11 from n1 to n1 in graph3
added edge e12 from n1 to n2 in graph3
added edge e13 from n1 to n3 in graph3
added edge e21 from n2 to n1 in graph3
added edge e22 from n2 to n2 in graph3
added edge e23 from n2 to n3 in graph3
added edge e31 from n3 to n1 in graph3
added edge e32 from n3 to n2 in graph3
added edge e33 from n3 to n3 in graph3

# Print the nodes in the graph 
graph3 contains: n1 n2 n3
the children of n1 in graph3 are: n1(e11) n2(e12) n3(e13)
the children of n2 in graph3 are: n1(e21) n2(e22) n3(e23)
the children of n3 in graph3 are: n1(e31) n2(e32) n3(e33)
